### ISSUE TITLE
Password protection on web interface

### ISSUE THREAD

**Issue by MattDralle** [buildbotics/bbctrl-firmware#133]
It would be nice to be able to password protect the web page interface.

**Comment by jcoffland** [comment]
Yes, good idea.  I had been thinking about making the configuration password protected.  There could perhaps be two levels of security.  One which locks out all access without a password and another which only locks access to the configuration.

**Comment by MattDralle** [comment]
At 03:38 PM 5/9/2018 Wednesday, you wrote:

>Yes, good idea. I had been thinking about making the configuration password protected. There could perhaps be two levels of security. One which locks out all access without a password and another which only locks access to the configuration.


Sounds like a solid idea plan.

Matt





Matt G Dralle | Matronics | 581 Jeannie Way | Livermore | CA | 94550
925-606-1001 V | 925-606-6281 F | dralle@matronics.com Email
http://www.matronics.com/ WWW | Featuring Products For Aircraft
